logstash-core/lib/logstash/api/lib/app/service.rb

```diff
@@ -40,4 +44,15 @@ def get(key)
     end
     LogStash::Json.dump(data)
   end
+
+  private
+
+  def has_counters?
```

Comment:
That could be a good idea, but I'm more on the side of letting the collector notify when he has received events, or any other side of information we aim for. On the other side, I'm starting to believe we should cleanup all this real/dummy plugins and move to a unified solution here.